Block property files: cleaned up trailing whitespace & fixed some typos

SuperGeniusZeb opened the pull request:
The trailing whitespace (3 spaces) occurred with every instance of the
`hitbox` object. Must have been on the first instance that was
copy-pasted into all the others.

Also threw in a correction of a misspell of `wooden_pressure_plate.json` as `wooden_preassure_plate.json`.

**EDIT:** I've reverted the tabification of the files and fixed 2 missing commas in `tripwire.json`.

**EDIT 2:** A formal list of all net changes in this pull request:
* Fixed misspellling of `wooden_pressure_plate.json` as `wooden_preassure_plate.json`.
* Removed all trailing whitespace from multiple instances of the `hitbox` object in the files.
* There were 2 commas missing in `twipwire.json`, and in the same spot, 2 pairs of bunched-up brackets like this -> `}{`. These typos have been fixed.
* In the `blocks_format.json` template, there were several instances of `"`s inside `""`s. this causes syntax errors and makes it harder to spot other typos/mistakes when editing the file, so I fixed that by changing the inside `"`s to `'`s instead. For example:
`"STRING: if it's capable of burning it'll also burn up, unless the value is "ever", "no" (default) means cannot burn, "yes" means can burn",`
was changed to
`"STRING: if it's capable of burning it'll also burn up, unless the value is 'ever', 'no' (default) means cannot burn, 'yes' means can burn",`
* There were also some inconsistencies in `blocks_format.json` where sometimes when one of the valid string arguments were listed, they would be listed as `name` instead of `'name'`. They now all use `'`s.

FVbico commented:
I kinda prefer spaces; and that is more consistent with the files mc already has.

SuperGeniusZeb commented:
The tabification has now been reverted and another typo fix has been made for `tripwire.json`.

SuperGeniusZeb commented:
A formal list of all net changes in this pull request:
* Fixed misspellling of `wooden_pressure_plate.json` as `wooden_preassure_plate.json`.
* Removed all trailing whitespace from multiple instances of the `hitbox` object in the files.
* There were 2 commas missing in `twipwire.json`, and in the same spot, 2 pairs of bunched-up brackets like this -> `}{`. These typos have been fixed.
* In the `blocks_format.json` template, there were several instances of `"`s inside `""`s. this causes syntax errors and makes it harder to spot other typos/mistakes when editing the file, so I fixed that by changing the inside `"`s to `'`s instead. For example:
`"STRING: if it's capable of burning it'll also burn up, unless the value is "ever", "no" (default) means cannot burn, "yes" means can burn",`
was changed to
`"STRING: if it's capable of burning it'll also burn up, unless the value is 'ever', 'no' (default) means cannot burn, 'yes' means can burn",`
* There were also some inconsistencies in `blocks_format.json` where sometimes when one of the valid string arguments were listed, they would be listed as `name` instead of `'name'`. They now all use `'`s.
